src/otx/core/model/entity/classification.py

```diff
@@ -74,14 +74,66 @@ def head_forward_fn(self, x: torch.Tensor) -> torch.Tensor:
         output = neck(x)
         return head([output])
 
-    def remove_explain_hook_handle(self) -> None:
-        """Removes explain hook from the model."""
-        if self.explain_hook.handle is not None:
-            self.explain_hook.handle.remove()
+    # def remove_explain_hook_handle(self) -> None:
+    #     """Removes explain hook from the model."""
+    #     if self.explain_hook.handle is not None:
+    #         self.explain_hook.handle.remove()
+
+    # def reset_explain_hook(self) -> None:
+    #     """Clear all history of explain records."""
+    #     self.explain_hook.reset()
+
+    def forward_explain(
+        self,
+        inputs: T_OTXBatchDataEntity,
+    ) -> T_OTXBatchPredEntity | OTXBatchLossEntity:
+        """Model forward function."""
+        self.model.explain_fn = self.get_explain_fn()
+
+        # If customize_inputs is overridden
+        outputs = (
+            self._forward_explain_image_classifier(self.model, **self._customize_inputs(inputs))
+            if self._customize_inputs != OTXModel._customize_inputs
+            else self.model(inputs)
+        )
+        return (
+            self._customize_outputs(outputs, inputs)
+            if self._customize_outputs != OTXModel._customize_outputs
+            else outputs
+        )
+
+    @staticmethod
+    def _forward_explain_image_classifier(
```

Comment:
Which forward will we use for export? forward_explain can not be exported, so we should use a low level one

Reply:
`_forward_explain_image_classifier` will be used for export